caab-api/open-api-specification.yml

```diff
@@ -381,19 +480,42 @@ paths:
           description: 'Not found'
         '500':
           description: 'Internal server error'
-  /applications/{id}/linked-cases/{linked-case-id}:
+  /linked-cases/{linked-case-id}:
     delete:
       tags:
-        - applications
+        - linked cases
       summary: 'Remove a linked case from an application'
       operationId: 'removeApplicationLinkedCase'
```

Comment:
Rename operation to removeLinkedCase, now that it's not application-scoped?

Reply:
updated